projects/kit/src/lib/masks/time/utils/stringify-time.ts

```diff
@@ -0,0 +1,45 @@
+import {DEFAULT_TIME_SEGMENT_MAX_VALUES} from '../../../constants';
+import type {MaskitoTimeSegments} from '../../../types';
+import {padTimeSegments} from '../../../utils/time';
+import type {MaskitoTimeParams} from '../maskito-time-params';
+
+export function maskitoStringifyTime(
+    milliseconds: number | null,
+    {
+        mode,
+        timeSegmentMaxValues = {},
+    }: Pick<MaskitoTimeParams, 'mode' | 'timeSegmentMaxValues'>,
+): string {
+    if (milliseconds === null) {
+        return '';
+    }
+
+    const maxValues: MaskitoTimeSegments<number> = {
+        ...DEFAULT_TIME_SEGMENT_MAX_VALUES,
+        ...timeSegmentMaxValues,
+    };
+
+    const millisecondsInSecond = maxValues.milliseconds + 1;
+    const millisecondsInMinute = (maxValues.seconds + 1) * millisecondsInSecond;
+    const millisecondsInHour = (maxValues.minutes + 1) * millisecondsInMinute;
```

Comment:
```suggestion
    const msInSecond = maxValues.milliseconds + 1;
    const msInMinute = (maxValues.seconds + 1) * millisecondsInSecond;
    const msInHour = (maxValues.minutes + 1) * millisecondsInMinute;
```

Reply:
Done.

Reply:
@demensky check this comment again, please. Actually, it is not done

Reply:
Yes you are right. I changed it in another file but forgot in this one. Done.